Makefile

```diff
@@ -3,11 +3,15 @@
 ifeq ($(wildcard .git/*),)
 $(warning semver disabled - building from release zip)
 GITCOMMIT := ""
+GITSHA := ""
 GITDATE := ""
+BRANCH := ""
 SEMVER := $(shell basename $(CURDIR))
 else
 GITCOMMIT := $(shell git rev-parse --short HEAD)
 GITDATE := $(shell git log -1 --format=%cd --date=unix)
+GITSHA := $(shell git rev-parse HEAD)
+BRANCH := $(shell git rev-parse --abbrev-ref HEAD | sed 's/[^[:alnum:]\.\_\-]/-/g')
```

Comment:
Note: this sanitizes the branch name to conform with docker tagging syntax rules.